Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -4163,7 +4166,7 @@
 					};
 					A7D1F9441C850B7C000D41D5 = {
 						CreatedOnToolsVersion = 7.2.1;
-						DevelopmentTeam = 48YBP49Y5N;
+						DevelopmentTeam = 5DAN4UM3NC;
```

Comment:
Hey let's not commit these changes, it might confuse CircleCI 👍 I'm guessing this was to get it to build locally on device?

Reply:
Reverting now. These changes were commited by accident :)